website/src/components/Stats/Stats.components.tsx

```diff
@@ -38,30 +39,191 @@ type MessageTreeStateStatsProps = {
   };
 };
 
+function getTableData(
+  stat: StatType & {
+    stats: {
+      lang: string;
+      state: string;
+      count: number;
+    }[];
+  },
+  t
+) {
+  // The headers come from the message tree State.
+  // These two states are not considered because they make
+  // the table too wide and they are not so interesting:
+  // - "ready_for_scoring",
+  // - "scoring_failed"
+  let headers = [
```

Comment:
Maybe we could declare this outside of the function and rename to something like `headingsOrder` I assume that is why we declare it? Then you could map over this and declare `headings` as a `const` below.

Reply:
Done.